inc/dg/helmholtz.h

```diff
@@ -179,12 +179,26 @@ struct Helmholtz2
      * @param jfactor The jfactor used in the Laplace operator (probably 1 is always the best factor but one never knows...)
      * @note The default value of \f$\chi\f$ is one
      */
-    Helmholtz2( const Geometry& g, bc bcx, bc bcy, double alpha = 1., direction dir = dg::forward, double jfactor=1.):
-        laplaceM_(g, bcx,bcy,normed, dir, jfactor), 
-        temp1_(dg::evaluate(dg::one, g)), temp2_(temp1_),chi_(temp1_),
-        alpha_(alpha)
+    Helmholtz2( const Geometry& g, bc bcx, bc bcy, double alpha = 1., direction dir = dg::forward, double jfactor=1.)
     { 
+              construct( g, bcx, bcy, alpha, dir, jfactor);
     }
+    ///@copydoc Helmholtz::Helmholtz2(const Geometry&,bc,bc,double,direction,double)
+    void construct( const Geometry& g, bc bcx, bc bcy, double alpha = 1, direction dir = dg::forward, double jfactor = 1.) 
+    {
+        laplaceM_.construct( g, bcx, bcy, dg::normed, dir, jfactor);
+        dg::blas1::transfer( dg::evaluate( dg::one, g), temp1_);
+	dg::blas1::transfer( dg::evaluate( dg::one, g), temp2_);
+        alpha_ = alpha; 
+    }
+    ///@copydoc Helmholtz::Helmholtz2(const Geometry&,double,direction,double)
+    void construct( const Geometry& g, double alpha = 1, direction dir = dg::forward, double jfactor = 1.) 
+    {
+        laplaceM_.construct( g, dg::normed, dir, jfactor);
+        dg::blas1::transfer( dg::evaluate( dg::one, g), temp1_);
+	dg::blas1::transfer( dg::evaluate( dg::one, g), temp2_);
+        alpha_ = alpha; 
+    }    
```

Comment:
In c++11 kann man die Constructor wohl einfacher schreiben, aber fürs erste können wir das mal so lassen